frontend/src/features/mapSideBar/SidebarContents/subforms/ParcelIdentificationForm.scss

```diff
@@ -52,7 +56,8 @@
       .form-row {
         display: flex;
         padding: 0px 20px;
-        input,
+        > input,
```

Comment:
more specific targeting of input fields to avoid unintentional input length limits.